data/users/externaloidc.py

```diff
@@ -143,16 +145,16 @@ def sync_oidc_groups(self, user_groups, user_obj):
                         f"External OIDC Group Sync: Cannot retrieve quay team synced with the oidc group: {oidc_group}"
                     )
 
+                if team.user_exists_in_team(user_obj, team_synced.team):
+                    continue
+
                 # add user to team
                 try:
                     team.add_user_to_team(user_obj, team_synced.team)
-                except InvalidTeamException as err:
+                except Exception as err:
```

Comment:
We don't want to catch general exceptions, as this won't surface real errors. We can keep the except block as it was before